File: bench/benchmarks/trial_primes/trial_primes.js
const N = 200000;
let count = 0;
for (let n = 2; n <= N; n++) {
    let isPrime = true;
    for (let d = 2; d * d <= n; d++) {
        if (n % d === 0) {
            isPrime = false;
            break;
        }
    }
    if (isPrime) count++;
}
console.log(count);

File: bench/benchmarks/trial_primes/trial_primes.py
N = 200000
count = 0
for n in range(2, N + 1):
    is_prime = True
    d = 2
    while d * d <= n:
        if n % d == 0:
            is_prime = False
            break
        d += 1
    if is_prime:
        count += 1
print(count)
